SCClassLibrary/scide_scqt/ScIDE.sc

```diff
@@ -629,6 +633,12 @@ Document {
 	}
 
 	close { ScIDE.close(quuid); }
+	
+	save { |docPath|
+		docPath = docPath ? path;
+		if(docPath.isNil, {MethodError("Document saved requires specified path\n", this).throw; });
```

Comment:
`if(docPath.isNil) { MethodError("Document saved requires specified path", this).throw };`

see https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#spaces-in-expressions-and-statements

no newline needed in the error string, at least AFAIK

typically we prefer to notate `if`s like that in the class library, although we don't have a specific rule for that. open to preference i guess

Reply:
> typically we prefer to notate `if`s like that in the class library, although we don't have a specific rule for that. open to preference i guess

I'm a bit old school I guess. I think in the earlier versions only ```if(conditon, {})``` was allowed in that style. I still prefer it implicitly as it makes clearer that it's a method call, but I accept that's esoteric. ;-) But then I was happy with ```boolean.if(...)``` for this reason though many consider it an abomination, LOL.